content/en/dashboards/widgets/geomap.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Geomap Widget
+kind: documentation
+widget_type: "geomap"
```

Comment:
this has to map to the api spec - https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/dashboard.yaml - which doesn't look like it includes this one yet. Do you know if there's a plan to add it?

Reply:
I'm pinging the team to add it